tests/php/includes/admin/helper/class-tests-wc-helper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Class WC_Tests_WC_Helper file.
+ *
+ * @package WooCommerce|Tests|WC_Helper.
+ */
+
+/**
+ * Class WC_Tests_WC_Helper.
+ */
+class WC_Tests_WC_Helper extends WC_Unit_Test_Case {
```

Comment:
Since this is a test for legacy code in the `includes` directory, this test actually belongs in the `legacy` test directory!